data/mods/My_Sweet_Cataclysm/sweet_harvest.json

```
@@ -56,5 +56,22 @@
     "leftovers": "ruined_candy",
     "message": "As you butcher the corpse, it turns into a sugary mush of licorice.",
     "entries": [ { "drop": "licorice", "type": "flesh", "mass_ratio": 0.7 } ]
+  },
+  {
+    "id": "pumpkin",
+    "type": "harvest",
+    "leftovers": "ruined_candy",
+    "message": "As you butcher the corpse, it turns into a mess of various pumpkin parts.",
+    "entries": [
+      { "drop": "pumpkin", "type": "flesh", "mass_ratio": 0.6 },
+      { "drop": "seed_pumpkin", "base_num": [ 3, 5 ], "max": 5, "type": "offal" }
+    ]
+  },
+  {
+    "id": "cane_hulk",
+    "type": "harvest",
+    "leftovers": "ruined_candy",
+    "message": "As you butcher the corpse, it turns into a pile of candy canes.",
```

```suggestion
    "message": "You break the corpse into smaller candy canes.",
```
It deosn't magically turn into candy canes, it's literally made of candy canes